src/inference/src/cpp_interfaces/interface/ie_iplugin_internal.cpp

```diff
@@ -37,7 +37,7 @@
 #include "openvino/core/runtime_attribute.hpp"
 #include "openvino/op/util/op_types.hpp"
 #include "openvino/pass/manager.hpp"
-#include "threading/ie_executor_manager.hpp"
+#include "openvino/runtime/threading/executor_manager.hpp"
 #include "transformations/utils/utils.hpp"
```

Comment:
This file is legacy, and will be removed in 24.0, not need to change it.

Reply:
ok, updated.